src/controller/ReadInteraction.h

```diff
@@ -53,6 +54,13 @@ CHIP_ERROR ReportAttribute(Messaging::ExchangeManager * exchangeMgr, EndpointId
     readParams.mpAttributePathParamsList    = readPaths.get();
     readParams.mAttributePathParamsListSize = 1;
 
+    auto dataVersionFilters = Platform::MakeUnique<app::DataVersionFilterParams>(endpointId, clusterId, aDataVersion);
```

Comment:
When we have no data version, we shoud not be allocating this at all, no?

Reply:
agree. fixed, thanks